docs/yarn.lock

```diff
@@ -0,0 +1,244 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why are you removing this from gitignore and adding it here?

Reply:
This is needed by the `@actions/setup-node` action.

Reply:
Actually the action only needs to root lockfile, I can remove that one.

Reply:
As discussed offline, we can make sure this file is constantly updated with Dependabot.